app/assets/appfilter.xml

```diff
@@ -1,26 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <resources>
-
```

Comment:
I test the same strings. It won't worked for me. I must add this `item
        component=":CALENDAR"
        drawable="calendar" />`

Reply:
Lawnchair didn't designed to support this. 
<item component=":CALENDAR" drawable="calendar" />
Can you check if the activity is correctly mentioned ? Latest dev installed ?Tried  retsrat lawnchir ?

It was working fine samsung calendar even without themed

Reply:
I can test it on the last ci build. I test it on kvaesitso

Reply:
Downloaded kvaesitso , dynamic icons applied to Samsung calendar.  Please check you calendar package and activity added here.


![WhatsApp Image 2022-11-17 at 11 17 20 PM](https://user-images.githubusercontent.com/7315975/202519889-de1ca150-a21d-46f7-ad17-afbf7013d321.jpeg)